src/History/inc/History.h

```diff
@@ -0,0 +1,68 @@
+#ifndef HISTORY_H
+#define HISTORY_H
+
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file History.h
+* @brief history function declarations.
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include <vector>
+#include <string>
+
+#include "flatbuffers/flatbuffers.h"
+#include "history_generated.h"
+
+namespace calculator {
+
+/**
+* @brief History interface which exposes to other components
+*/
+
+class History
+{
+    public:
+        /**
+         * @brief constructor for History
+        */
+        History();
+
+        /**
+         * @brief creates a vector out of a string
+         * @param [in] fullResult string of the full result taken from result object ex: "1+1 = 2"
+        */
+        void addToHistory(std::string fullResult);
```

Comment:
this should likely use a rvalue reference `std::string &&fullResult` -- look into move sematics

Reply:
I ended up just making it a const &, since I use the full result value in a cout right after this is called, I think If I used the move sematics, that value wouldn't be there after it was moved.